torch/fx/experimental/_dynamism.py

```diff
@@ -29,14 +29,21 @@ def module_to_nested_dict(module: torch.nn.Module) -> dict[str, Any]:
     self_dict["_modules"] = {}
 
     for attr_name in dir(module):
-        if not attr_name.startswith("_") and not callable(getattr(module, attr_name)):
-            attr_value = getattr(module, attr_name)
-            if (
-                not isinstance(attr_value, torch.nn.Module)
-                and isinstance(attr_value, (int, float, torch.Tensor))
-                and type(attr_value) is not bool
+        try:
+            if not attr_name.startswith("_") and not callable(
+                getattr(module, attr_name)
             ):
-                self_dict[attr_name] = attr_value
+                attr_value = getattr(module, attr_name)
```

Comment:
I wonder where exaclty is the error raised? when you do getattr(module, attr_name) ?

Reply:
It's a @property decorator that makes it so we invoke the funciton and raise when we access it